src/Corporation/ui/modals/IssueDividendsModal.tsx

```diff
@@ -19,7 +19,7 @@ export function IssueDividendsModal(props: IProps): React.ReactElement {
   const corp = useCorporation();
   const [percent, setPercent] = useState(0);
 
-  const canIssue = !isNaN(percent) && percent >= 0 && percent <= CorporationConstants.DividendMaxPercentage * 100;
+  const canIssue = !isNaN(percent) && percent >= 0 && percent <= CorporationConstants.DividendMaxRate * 100;
```

Comment:
(i think)
```suggestion
  const canIssue = !isNaN(percent) && percent >= 0 && percent <= CorporationConstants.DividendMaxRate;
```

Reply:
and obviously update to be rate not % or did you actualy want % here which is weird why it was being *100 to begin with